src/qrcode/utils.ts

```diff
@@ -126,9 +127,10 @@ export function* handleSecureSend(
 }
 
 function* extractQRAddressData(qrCode: QrCode) {
-  // Regex matches any 40 hexadecimal characters prefixed with "0x" (case insensitive)
-  if (/^0x[a-f0-9]{40}$/gi.test(qrCode.data)) {
-    qrCode.data = `celo://wallet/pay?address=${qrCode.data}`
+  // strip network prefix if present
+  const qrAddress = qrCode.data.replace(/^.*:/, '')
```

Comment:
```suggestion
  const qrAddress = qrCode.data.split(':').at(-1)
```
This is a bit faster than the regex and has the added benefit of not being regex.

Reply:
+1 on the "not being a regex".  should we check length of array === 2?  to avoid unforeseen stuff like `celo:0xeee:0xfff` (no idea what the might mean, but you get the idea -- kinda a halfhearted suggestion).

Reply:
I'm pretty sure thats not something we'll have to worry about. I believe most of these URL's are based on [ERC-681](https://eips.ethereum.org/EIPS/eip-681) which doesn't include any additional colons